day1/part2.py

```diff
@@ -0,0 +1,43 @@
+with open('input.txt', 'r') as f:
+    numbers = ('one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine')
+    lines = list(map(str.rstrip, f.readlines()))
+    num_of_line = 0
+    counter = 0
+    for line in lines:
+        counter += 1
+        for character in line:
```

Comment:
Same code twice or more times (2 + 2). 😃 
Try to keep your code DRY meaning do not repeat yourself. 💯